apps/full-site-editing/package.json

```diff
@@ -46,5 +47,18 @@
 	},
 	"dependencies": {
 		"classnames": "2.2.6"
+	},
+	"wp-env": {
+		"plugin-dir": "full-site-editing-plugin",
+		"plugin-name": "Full Site Editing Plugin",
+		"docker-template": "./bin/docker-compose.override.yml.template",
+		"welcome-logo": [
```

Comment:
The important part! :)

Reply:
also, if anyone has ideas on a better ASCII logo, feel free to add it. I just pulled it off an ascii text generator website